utils/test-utils/src/jvmMain/kotlin/io/kinference/utils/TestRunner.kt

```diff
@@ -1,12 +1,12 @@
 package io.kinference.utils
 
-import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.*
 import kotlinx.coroutines.runBlocking
 
 actual object TestRunner {
     actual fun runTest(platform: Platform?, block: suspend CoroutineScope.() -> Unit) {
         if (platform == null || platform == Platform.JVM) {
-            runBlocking { block() }
+            runBlocking(Dispatchers.Default) { block() }
```

Comment:
Why Dispatchers.Default?

Reply:
Because if we want testing parallel code, then we need `Dispatchers.Default`